ctapipe/plotting/charge_resolution.py

```diff
@@ -0,0 +1,360 @@
+import numpy as np
+import pandas as pd
+from matplotlib.ticker import FuncFormatter
+from matplotlib import pyplot as plt
+import os
+from ctapipe.core import Component
+from traitlets import Unicode, Int
+
+
+def sum_errors(array):
+    """
+    Simple sum of squares to combine errors
+
+    Parameters
+    ----------
+    array : ndarray
+
+    Returns
+    -------
+    float
+    """
+    return np.sqrt(np.sum(np.power(array, 2)) / array.size)
+
+
+def bin_dataframe(df, n_bins):
+    """
+    Assign a "bin" column to the dataframe to indicate which bin the true
+    charges belong to.
+
+    Bins are assigned in log space.
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+    n_bins : int
+        Number of bins to allow in range
+
+    Returns
+    -------
+    pd.DataFrame
+    """
+    true = df['true'].values
+    min_ = true.min()
+    max_ = (true.max() // 500 + 1) * 500
```

Comment:
Would it be possible to explain the "magic number" `500` in the line above?

Reply:
I will remove this "magic number" tweaking. I don't think theres any reason for it, I just missed it.

Reply:
Cool thx